t/broadcast.t

```diff
@@ -39,10 +38,13 @@ __DATA__
             ngx.log(ngx.ERR, "failed to init_worker events: ", err)
         end
 
+        local i = 0
+
         ev:subscribe("*", "*", function(data, event, source, wid)
-            ngx.log(ngx.DEBUG, "worker-events: handler event;  ","source=",source,", event=",event, ", wid=", wid,
-                    ", data=", data)
-                end)
+            i = i + 1
```

Comment:
Seems the `i` here is used only once. I assume it is for test style consistency.